dashboard/scripts/test_seed

```diff
@@ -3,13 +3,11 @@
 # test the seed:all rake task (mostly that it is idempotent)
 
 SQL="select id, \`key\` from videos; \
-     select id, name, video_id from concepts; \
```

Comment:
Updated (PTAL). And I did know of this dependency (just forgot), I'm pretty sure you mentioned it as part of your cleanup. Thanks for the good eyes!